app/models/Departure.scala

```diff
@@ -97,4 +97,6 @@ object Departure {
         (__ \ "notificationBox").writeNullable[Box]
     )(unlift(Departure.unapply))
 
+  implicit def formatsDeparture(implicit write: Writes[LocalDateTime]): OFormat[Departure] =
+    OFormat(readsDeparture, writesDeparture)
```

Comment:
This `implicit write` is weird but incompatibly-encoded date values are written if I remove it, i.e. they can be written but can't be read by the current JSON Reads for Message.